contracts/libraries/LibRealm.sol

```diff
@@ -66,6 +67,63 @@ library LibRealm {
     }
   }
 
+  function placeTile(
+    uint256 _realmId,
+    uint256 _tileId,
+    uint256 _x,
+    uint256 _y
+  ) internal {
+    AppStorage storage s = LibAppStorage.diamondStorage();
+    uint8[5] memory widths = [
+      8, //humble
```

Comment:
Cleaner to make these constants to they can be shared by tile and installation functions